intelhex/__init__.py

```diff
@@ -863,9 +863,9 @@ def merge(self, other, overlap='error'):
             raise TypeError('other should be IntelHex object')
         if other is self:
             raise ValueError("Can't merge itself")
-        if overlap not in ('error', 'ignore', 'replace'):
+        if overlap not in ('error', 'ignore', 'replace', 'check'):
```

Comment:
Please amend the method's documentation a couple lines higher as well

Reply:
I agree that identical is clearer than check (even if it's not a verb as for other methods).